README.md

```diff
@@ -26,7 +26,7 @@ The OpenTrials API service.
 2. Run `npm install`;
 3. Run `npm run migrate`;
 4. (Optional) If you want, you can add some seed data using `npm run seed`;
-5. Run `npm run reindex`;
+5. (If you ran 4) Run `npm run reindex`;
```

Comment:
I'm not completely sure if our endpoints would fail if there's no indexes on ES, but +1 for avoiding exception from reindex if there's no data on database